src/views/eip3643/EIP3643.tsx

```diff
@@ -63,10 +64,10 @@ export default function EIP3643() {
             <NFT />
           </TabPanel>
           <TabPanel>
-            <ManageRegistry isAgents={false} />
+            <ManageRegistry isAgents={false} identities={identities} />
```

Comment:
we can grab `identities` from context inside the `ManageRegistry`, no need to pass them from the parent I think